mne/source_estimate.py

```diff
@@ -1583,6 +1583,98 @@ def in_label(self, label):
             subject=self.subject,
         )
         return label_stc
+    
+    def save_as_surface(self, fname, src, scale=1, scale_rr=1e3):
+        """Save a surface source estimate (stc) as a GIFTI file.
+
+        Parameters
+        ----------
+        fname : path-like
+            filename basename to save files as
+            Will write anatomical gifti plus time series gifti for both lh/rh
+            ex: 
+                "basename" --> "basename.lh.gii" and "basename.lh.time.gii"
+                           --> "basename.rh.gii" and "basename.rh.time.gii"
```

Comment:
Sphinx and reST probably won't like the spacing here, might as well just do
```suggestion
            Will write anatomical gifti plus time series gifti for both lh/rh,
            for example ``"basename"`` will write ``"basename.lh.gii"``, 
            ``"basename.lh.time.gii"``, ``"basename.rh.gii"``, and
            ``"basename.rh.time.gii"``.
```